src/core/execute.c

```diff
@@ -111,6 +111,10 @@
 
 #define SNDBUF_SIZE (8*1024*1024)
 
+/* Default delegated cgroup suffices */
+#define DELEGATE_CGROUP_CONTROL ".control"
+#define DELEGATE_CGROUP_PAYLOAD ""
```

Comment:
i think this doesn't make the code either to grok really. in particular the fact that DELEGATE_CGROUP_PAYLOAD is in fact *not* a valid cgroup name is utterly confusing.

macros are great for making stuff configurable at build time, or to simply complex expressions, or to make things compiler errors that are otherwise easily mistyped and never seen.

But I am not convinced either of this is the case here, and it would be a lot simpler to just spell out the group names literally, as before.